src/app/workout/[workoutId]/page.tsx

```
@@ -0,0 +1,273 @@
+  CalendarDaysIcon, 
+  PencilIcon, 
+  TrashIcon,
+  StarIcon as StarOutline,
+  ClockIcon,
+  CheckIcon
+} from '@heroicons/react/24/outline';
+import { StarIcon as StarSolid } from '@heroicons/react/24/solid';
+import { useWorkout } from '@/lib/hooks/useWorkout';
+import { useProfile } from '@/lib/hooks/useProfile';
+import { useToggleFavorite, useDeleteWorkout } from '@/lib/hooks/useMutations';
+import { Exercise, Set, Workout } from '@/types/workout';
```

Biome (1.9.4)

[error] Do not shadow the global "Set" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)